cpp/.gitignore

```diff
@@ -0,0 +1,4 @@
+/build_meson/
+/subprojects/Catch2-2.5.0/
+/subprojects/packagecache/
+/cmake-build-debug/
```

Comment:
A tiny point.... I have my CLion configured to create some other configurations as well, like `cmake-debug-release` - so would really appreciate this line being changed to:
`/cmake-build-*/`

Reply:
Great point, I'm not used to Clion yet.